src/plugins/autonomy/MotorSchemas/MotorSchemas.cpp

```diff
@@ -75,6 +75,16 @@ void MotorSchemas::init(std::map<std::string, std::string> &params) {
     show_shapes_ = sc::get("show_shapes", params, false);
     pub_vel_vec_ = sc::get("pub_vel_vec", params, false);
     max_speed_ = sc::get<double>("max_speed", params, 21);
+    
+    // Get Max Current Speed and adjust autonomy to be able to go 110% of Max Current
+    // ("GlobalNetwork", "Force_Field")
+    // auto msg_force = std::make_shared<sc::Message<Eigen::Vector3d>>();
+    // sc::sensor::SideScanSonarSensorType SSS_msg_;
+
+    auto callback_force = [&] (auto &msg) {
+            Force_ = msg->data;
+        };
+    subscribe<Eigen::Vector3d>( "GlobalNetwork", "Force_Field", callback_force);
```

Comment:
I'm not sure the MotorSchemas behavior would have access to the external force measurement through a real sensor.

Reply:
Eliminated change to keep MotorSchemas clean of project specific modifications.